src/main/java/fi/hsl/transitdata/vehicleposition/application/VehiclePositionHandler.java

```diff
@@ -182,18 +182,14 @@ public void handleMessage(Message message) {
                     throw x;
                 }
                 
-                String detailMessage = "";
-                
                 try {
                     StopStatusProcessor.StopStatus stopStatus = stopStatusProcessor.getStopStatus(data);
-                    detailMessage = "stopStatusProcessor.getStopStatus(data) called";
                     String uniqueVehicleId = getUniqueVehicleId(data.getTopic().getOperatorId(), data.getTopic().getVehicleNumber());
-                    detailMessage = "getUniqueVehicleId called";
                     PassengerCount.Payload passengerCount = passengerCountCache.getPassengerCount(uniqueVehicleId, data.getPayload().getRoute(), data.getPayload().getOday(), data.getPayload().getStart(), data.getPayload().getDir());
-                    detailMessage = "passengerCountCache.getPassengerCount called";
+                    
                     if (!isValidPassengerCountData(passengerCount)) {
                         if (passengerCount != null) {
-                            log.warn("Passenger count for vehicle {} was invalid (vehicle load: {}, vehicle load ratio: {})",
+                            log.debug("Passenger count for vehicle {} was invalid (vehicle load: {}, vehicle load ratio: {})",
```

Comment:
I don't know what the process is for catching bugs in the onboard APC devices. Would turning this into debug effectively silence it? Do we have a feedback loop elsewhere to communicate with the APC device vendor?

Reply:
Or maybe it's not a bug for vehicleLoad to be negative as there is measurement error anyway?

Reply:
log.debug was used before I added logging for debugging purposes